### PR TITLE
fixing jshint score

### PR DESCRIPTION
**Pull request by tldavies**

### CODE REVIEW

**Comment by heckj** [comment]
+1

**Comment by benbp** [comment]
:+1:

**Comment by tldavies** [comment]
Coveralls is red cause code coverage went down.

File: lib/services/ws-service.js [hunk 1]
```diff
@@ -37,7 +37,8 @@ function webSocketServiceFactory(
             this.error(new WebSocketError('Broadcast failed, no server is available.'));
         }
 
-        this.webSocketServer.clients.forEach(wsClientBroadcast.bind(this));
+        var wsService = this;
```
Comment (benbp) [inline lib/services/ws-service.js]: @rolandpoulter this change ok, or does `this` get bound differently in the closure? I've caused a lot of subtle scoping bugs by doing things like this, so perhaps we just want to add an `ignore: true` line or something?

Comment (rolandpoulter) [inline lib/services/ws-service.js]: This is fine for this case. The broadcast method isn't being used yet anyways.

**Comment by tldavies** [comment]
@RackHD/corecommitters

**Pull request merged**